lib/memjs/memjs.js

```diff
@@ -355,7 +356,7 @@ Client.prototype.flush = function(callback) {
     serv.write(seq, request);
   };
 
-  for (var i in this.servers) {
+  for (i = 0; i < this.servers.length; i++) {
```

Comment:
OIC, you declared it on line 336, but I still think it preferable to declare it inline each time.